### DISCUSSION TITLE
Search for 1.3.2 code

### DISCUSSION THREAD

**Discussion by rvs76** [spring-projects/spring-net#228] [Q&A]
For an old application, I search the 1.3.2 code. Anyone this code ?

**Comment by lahma** [comment]
Release have been tagged, you can find the code here: https://github.com/spring-projects/spring-net/tree/spring-net-1.3.2

**Comment by rvs76** [comment]
It's possible to have the signature file for Spring.Core.dll, Spring.Data.dll, Spring.Aop.dll and Spring.Data.NHibernate21.dll ?

**Reply by lahma** [reply]
I think you can calculate hashes you need from the NuGet package contents, there's no official signatures for this release I think.
